app/src/common/shared/com/igalia/wolvic/ui/views/TabsBarItem.java

```diff
@@ -150,7 +148,7 @@ public void onLocationChange(@NonNull WSession session, @Nullable String url) {
             return;
         }
 
-        if (url == null) {
+        if (url == null || UrlUtils.isAboutPage(url)) {
```

Comment:
Why this ?

Reply:
It removes the URL from the tab widget, otherwise it appears as `"newtab"` (because the protocol prefix is stripped).

The favicon is empty because web icons come from a Firefox component and by default local pages don't get one.

We can add our own icon here if we want.